src/docs/sanbase/social-trends-search/index.md

```diff
@@ -2,43 +2,38 @@
 title: Social Tool
```

Comment:
Lets change the name of that article from `social-trends-search` to `social tool`